### ISSUE TITLE
[Question]: Document analysis with pictures and texts

### ISSUE THREAD

**Issue by lvyoudashuju** [infiniflow/ragflow#3343]
### Describe your problem

How to parse a file with both pictures and text, for example, a file contains a text and a picture, and the picture is a pure picture

**Comment by KevinHuSh** [comment]
Is it a PDF file?
The pure picture will attach to image caption if it has.

**Comment by lvyoudashuju** [comment]
Yes. I just uploaded a pdf file with pictures and text, but it doesn’t seem right when parsed.
![企业微信截图_17313787235623](https://github.com/user-attachments/assets/58117148-8a89-4098-bd0c-7ef99f7a4c3b)
